plugins/packages/packages_renderer.rb

```diff
@@ -26,6 +26,61 @@ def do_render
     end
   end
 
+  def do_render_comparison(description1, description2, description_common)
```

Comment:
Assignment Branch Condition size for do_render_comparison is too high. [26.94/15]